src/components/CourseInfo/CourseInfo.jsx

```diff
@@ -0,0 +1,58 @@
+import { Link, useParams } from "react-router-dom";
+
+import dateFormater from "../../helpers/dateFormatter";
+import formatDuration from "../../helpers/formatDuration";
+import Button from "../../common/Button/Button";
+import Author from "../CreateCourse/components/Author";
+import { mockedCoursesList, mockedAuthorsList } from "../../constants";
```

Comment:
Ideally for the 2nd task you would get rid of mocked courses. Please double-check the description for [CourseInfo component](https://github.com/chacon44/React/pull/4) and also Jose's PR and comments, as he's made a similar incorrect assumption.

Reply:
agreed to ignore this and apply in the next task